api/client/client.go

```diff
@@ -3822,6 +3826,26 @@ func (c *Client) ListResources(ctx context.Context, req proto.ListResourcesReque
 	}, nil
 }
 
+// repackIdentityCenterAccountAssignment converts a wire-format IdentityCenterAccountAssignment
+// resource back into an identitycenterv1.AccountAssignment instance.
+func repackIdentityCenterAccountAssignment(src *proto.IdentityCenterAccountAssignment) types.ResourceWithLabels {
```

Comment:
Nit: Just a thought, can this maybe be moved to a more appropriate package where IC resources (or maybe `types`) and named `ICAccountAssignmentToResourceWithLabels`? We probably have other similar converters somewhere already, right? Like there's `types.LegacyTo153Metadata` you're using below.

Not a blocker though.